src/ModelData.h

```diff
@@ -0,0 +1,32 @@
+#ifndef MODELDATA_H
+#define MODELDATA_H
+
+#include <GL/glew.h>
+#include <glm/glm.hpp>
+#include <vector>
+
+class ModelData
+{
+public:
+	ModelData(std::vector<glm::vec3>* pos, std::vector<glm::vec2>* textureCoord, std::vector<glm::vec3>* normal);
+	ModelData();
+	const std::vector<glm::vec3>& getPos();
+	const std::vector<glm::vec2>& getTexCoord();
+	const std::vector<glm::vec3>& getNormal();
+	const std::vector<unsigned int>& getIndices();
```

Comment:
I should've mentioned it before, but using `const` at the end of getter functions is good as well. This promises that the function won't change the state of `this`.